apps/dashboard/app/javascript/path_selector/path_selector_data_table.js

```diff
@@ -207,6 +209,16 @@ export class PathSelectorTable {
 
   // filter the response from the files API to remove things like hidden files/directories
   filterFileResponse(data) {
+    let regex = undefined
+
+    try {
+      if (this.filePattern !== undefined) {
+        regex = RegExp(this.filePattern);
+      }
+    } catch {
+      alert("The regular expression provided for this path selector did not compile");
```

Comment:
this is calling `window.alert` not the `alert` method in the javascript. Maybe we need to rename that function lol?